force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -50,6 +50,20 @@
         <shortDescription>Attendance</shortDescription>
         <value>Attendance</value>
     </labels>
+    <labels>
+        <fullName>Save_Attendance_Success</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Successfully saved {0} attendance records.</shortDescription>
+        <value>Successfully saved {0} attendance records.</value>
+    </labels>
+    <labels>
+        <fullName>Save_Attendance_No_Updates</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>There were no edited attendance records to update.</shortDescription>
+        <value>There were no edited attendance records to update.</value>
+    </labels>
```

Comment:
@lbeller Hi, Lauren. Can I please get a review on these toast messages? The first one is based on others we have, and the second one is a new concept that I'm happy to talk through. Essentially, the user hits save, but they hadn't made any changes.

Reply:
@bethbrains Sure, here are some minor changes:

- Successfully saved {0} attendance records. = Saved {0} attendance records.
-There were no edited attendance records to update = There are no attendance records to update. --OR-- There are no updates to save.